### PR TITLE
fix: ErrorBoundarySharedProps's info type

### PR DESCRIPTION
**Pull request by manudeli**
**What**: fix ErrorBoundarySharedProps's info type

**Why**: ErrorBoundarySharedProps's info can follow ErrorInfo of react

**How**: connect ErrorBoundary.onError.info to React.ErrorInfo

**Checklist**:

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

### CODE REVIEW

File: src/types.ts
Comment (manudeli) [inline src/types.ts:18]: We can provide description of react ErrorInfo too.

```ts
    //
    // Error Interfaces
    // ----------------------------------------------------------------------
    interface ErrorInfo {
        /**
         * Captures which component contained the exception, and its ancestors.
         */
        componentStack: string;
    }
```

**Review by manudeli — COMMENTED** [review]
I left comment as my intention

**Comment by bvaughn** [comment]
Hey @manudeli 👋🏼 Can you please combine these PRs? I know it's subjective but I'd prefer to just review and talk about small changes like this on a single PR.

**Comment by manudeli** [comment]
These changes are really small, but I think they have different themes. Therefore, your request to combine them into one Pull Request seems a little sad for me, who has already divided the Pull Requests. Although you have merge rights and I understood your preference, but I would like to keep this PRs separatly like now without any my effort to combine these PRs if no problem for your approving like now.

Could you approve this if no problem please?

**Comment by bvaughn** [comment]
One of the reasons I'd prefer to group them– this PR now has a conflict that prevents it from being merged.

**Comment by manudeli** [comment]
> One of the reasons I'd prefer to group them– this PR now has a conflict that prevents it from being merged.

Aha, I understood why you prefer it. anyway I rebased it for now.

**Comment by manudeli** [comment]
I think it would have been better if I had edited [this Pull Request](https://github.com/bvaughn/react-error-boundary/pull/152)'s title as `fix: type ErrorBoundary props` and combine this Pull Request in one PR.
I misunderstood as you want to combine all 6 Pull Request in one 🤣

**Review by bvaughn — APPROVED** [review]

**Pull request merged**
